docs/File-Formats.md

```diff
@@ -202,8 +202,8 @@ Custom attributes:
 #STRING<TAB>STRING<TAB>NUMBER<TAB>STRING<TAB>NUMBER<TAB>...
 #1<TAB>1<TAB>1<TAB>1<TAB>1<TAB>
 PATIENT_ID<TAB>OS_STATUS<TAB>OS_MONTHS<TAB>DFS_STATUS<TAB>DFS_MONTHS<TAB>...
-PATIENT_ID_1<TAB>DECEASED<TAB>17.97<TAB>Recurred/Progressed<TAB>30.98<TAB>...
-PATIENT_ID_2<TAB>LIVING<TAB>63.01<TAB>DiseaseFree<TAB>63.01<TAB>...
+PATIENT_ID_1<TAB>1<TAB>17.97<TAB>1<TAB>30.98<TAB>...
```

Comment:
Are we missing ":DECEASED" "1:Recurred/Progressed"?

Reply:
Thanks, I forgot to update this table, will update this now.